packages/clerk-js/src/ui/components/PaymentSources/AddPaymentSource.tsx

```diff
@@ -195,6 +196,38 @@ const AddPaymentSourceForm = withCardStateProvider(
             rowGap: t.space.$3,
           })}
         >
+          {clerk.instanceType === 'development' && (
+            <Box
+              sx={t => ({
+                background: t.colors.$neutralAlpha50,
+                paddingInline: t.space.$2,
+                paddingBlock: t.space.$1x5,
+                borderRadius: t.radii.$md,
+                borderWidth: t.borderWidths.$normal,
+                borderStyle: t.borderStyles.$solid,
+                borderColor: t.colors.$neutralAlpha100,
+                display: 'flex',
+                flexDirection: 'column',
+                rowGap: t.space.$2,
+              })}
+            >
+              <Text variant='caption'>Test card information</Text>
```

Comment:
We need to specify a colorScheme on the text here. Probably body?

![Screenshot 2025-04-30 at 12 08 27 PM](https://github.com/user-attachments/assets/23ff5542-21a5-4787-a659-bd0ba0910297)